core/src/accounts_background_service.rs

```diff
@@ -22,18 +22,24 @@ impl AccountsBackgroundService {
         let exit = exit.clone();
         let t_background = Builder::new()
             .name("solana-accounts-background".to_string())
-            .spawn(move || loop {
-                if exit.load(Ordering::Relaxed) {
-                    break;
+            .spawn(move || {
+                let mut i = 0;
+                loop {
+                    i += 1;
+                    if exit.load(Ordering::Relaxed) {
+                        break;
+                    }
+                    let bank = bank_forks.read().unwrap().working_bank();
+
+                    bank.process_dead_slots();
+
+                    // Currently, given INTERVAL_MS, we process 1 slot/100 ms
+                    if i % 10 == 0 {
```

Comment:
This is tooo rough. But I intend a validator to scan its all appendvecs once (or twice) a day.